blaze-server/src/main/scala/org/http4s/server/blaze/BlazeBuilder.scala

```diff
@@ -8,17 +8,20 @@ import java.net.InetSocketAddress
 import java.nio.ByteBuffer
 import java.security.{KeyStore, Security}
 import javax.net.ssl.{KeyManagerFactory, SSLContext, SSLEngine, TrustManagerFactory}
+
```

Comment:
So am I just fighting a losing battle here on using Twitter's import style over IntelliJ's?

Reply:
No, I'm losing a battle with intellij. I'll fix it and maybe invest the time to make the auto formatter work for me... Sorry you need to keep reminding me. 🙁